frameworks/keyed/mikado/src/main.js

```diff
@@ -25,8 +24,4 @@ const view = new Mikado(document.getElementById("tbody"), item, {
     store[1] = tmp;
 })
 .route("remove", target => store.splice(view.index(target), 1))
-.route("select", target => {
-    store[selected]["selected"] = "";
-    store[selected = view.index(target)]["selected"] = "danger";
-})
-.listen("click");
+.route("select", target => view.refresh(view.selected = view.index(target)));
```

Comment:
Implementation looks pretty clean just was trying to understand this one. It's only update where action on the view happens instead of the store. What does that mean? Where is selected state stored?

Reply:
It is stored to the view instance, so different views with the same data can handle its own select state.

Reply:
Gotcha so it's a bit more like a React having it's own `useState` in the child component. Love to dig in to understand how rows get unselected at some point but cool stuff.